client/tests/integration/roles.rs

```diff
@@ -141,3 +141,34 @@ fn register_metadata_role() -> Result<()> {
     test_client.submit(register_role)?;
     Ok(())
 }
+
+#[test]
+fn unregistred_role_removed_from_account() -> Result<()> {
+    let (_rt, _peer, test_client) = <TestPeer>::start_test_with_runtime();
+    wait_for_genesis_committed(&vec![test_client.clone()], 0);
+
+    let role_id: RoleId = "root".parse().expect("Valid");
+    let alice_id: AccountId = "alice@wonderland".parse().expect("Valid");
```

Comment:
I'm not very sure why, but we use another style for id. We use `<SomeThing as Idenfifiable>::Id>` type, not `SomeThingId`

Reply:
Please, check for it also in other your changes